src/potfile.c

```diff
@@ -373,10 +373,13 @@ int potfile_remove_parse (hashcat_ctx_t *hashcat_ctx)
   const hashes_t       *hashes       = hashcat_ctx->hashes;
   const module_ctx_t   *module_ctx   = hashcat_ctx->module_ctx;
   const potfile_ctx_t  *potfile_ctx  = hashcat_ctx->potfile_ctx;
+  const user_options_t *user_options = hashcat_ctx->user_options;
 
   if (potfile_ctx->enabled == false) return 0;
 
   if (hashconfig->potfile_disable == true) return 0;
+  
+  if (user_options->keep_guessing == true) return 0;
```

Comment:
This would only change the behavior in case the user is using the --keep-guessing command line argument. There are however modules which automatically enable the keep-guessing flag automatically, such as 9720 and 18100 and others. Those would not be covered with this test. Please replace this test with the following and we should be good:

```
if (hashconfig->opts_type & OPTS_TYPE_PT_NEVERCRACK) return 0;
```

The reason why this will also cover is best explained here:

https://github.com/hashcat/hashcat/blob/master/src/interface.c#L347-L350